tests/testing_support/validators/validate_transaction_errors.py

```diff
@@ -58,20 +58,16 @@ def _validate_transaction_errors(wrapped, instance, args, kwargs):
         else:
             compare_to = sorted([e.type for e in captured])
 
-        assert expected == compare_to, "expected=%r, captured=%r, errors=%r" % (expected, compare_to, captured)
+        assert expected == compare_to, f"expected={expected!r}, captured={compare_to!r}, errors={captured!r}"
```

Comment:
I think this may have been unintentional (the `!r` after the variable name)

Reply:
That's actually correct, the !r is the same raw string conversion that %r was doing. I considered changing it to drop the !r but decided to leave the behavior exactly the same for such a sweeping set of changes.